sql/hive/src/main/scala/org/apache/spark/sql/hive/client/IsolatedClientLoader.scala

```diff
@@ -69,7 +69,7 @@ private[hive] object IsolatedClientLoader extends Logging {
             // If the error message contains hadoop, it is probably because the hadoop
             // version cannot be resolved.
             val fallbackVersion = if (VersionUtils.isHadoop3) {
-              "3.3.1"
+              "3.3.2"
             } else {
               "2.7.4"
```

Comment:
By the way,  can we read the hadoop version of the project configuration here?

Reply:
That sounds like independent improvement idea. Could you file a JIRA for that?

Reply:
> That sounds like independent improvement idea. Could you file a JIRA for that?

Yea, will try to do this.

Reply:
I'm not sure this is easy since in this case the Hadoop version specified via `hadoop.version` in `pom.xml` is customized and is not `3.3.2`, which is why it can't be fetched from Maven.